### DISCUSSION TITLE
How print the raw imap messege sent?

### DISCUSSION THREAD

**Discussion by prRZ5F4LXZ** [mjs/imapclient#509] [Q&A]
I want to check the raw imap message sent by imapclient. How can I set up for imapclient so that the python program will print out what raw messages are sent?

**Comment by mjs** [comment]
You should be able to see the raw IMAP traffic sent and received by IMAPClient and it's internal imaplib client by turning on debug logging as described here: https://imapclient.readthedocs.io/en/master/concepts.html#logging
